katsdpcal/control.py

```diff
@@ -1089,7 +1089,8 @@ def _reset_solution_stores(self):
             'B': solutions.CalSolutionStoreLatest('B'),
             'BCROSS_DIODE': solutions.CalSolutionStoreLatest('BCROSS_DIODE'),
             'G': solutions.CalSolutionStore('G'),
-            'G_FLUX': solutions.CalSolutionStore('G')
+            'G_FLUX': solutions.CalSolutionStore('G'),
+            'B_POINTING': solutions.CalSolutionStore('B')
```

Comment:
By adding a comma after each item, you avoid changes to the previous last line (i.e. the one with `G_FLUX`) just to add a comma there. In Python `[1, 2, 3,]` is the same as `[1, 2, 3]`.
```suggestion
            'B_POINTING': solutions.CalSolutionStore('B'),
```